doc/releases/changelog-dev.md

```diff
@@ -9,6 +9,10 @@
 * The higher order primitives in program capture can now accept inputs with abstract shapes.
   [(#6786)](https://github.com/PennyLaneAI/pennylane/pull/6786)
 
+* The `PlxprInterpreter` classes can now handle creating dynamic arrays via `jnp.ones`, `jnp.zeros`,
+  and `jnp.arange`.
+  [#6865)](https://github.com/PennyLaneAI/pennylane/pull/6865)
+
```

Comment:
Probably not the most used function but supporting `broadcast_in_dim_p` also allows usage of `jnp.full`.

Reply:
added to the changelog